rpc/js/src/peer.ts

```diff
@@ -75,9 +81,14 @@ export async function newPeerConnectionForClient(
     }
     try {
       await peerConnection.setLocalDescription();
-      negotiationChannel.send(
-        btoa(JSON.stringify(peerConnection.localDescription))
-      );
+      let description = {
+        sdp: peerConnection.localDescription?.sdp,
+        type: peerConnection.localDescription?.type,
+      };
+      if (priority) {
+        description.sdp = description.sdp + `a=x-priority:${priority}\r\n`;
```

Comment:
same issue https://github.com/viamrobotics/goutils/pull/215/files#r1372023046

```suggestion
        description.sdp = [description.sdp, `a=x-priority:${opts.webrtcOptions?.priority}\r\n`].join("");
```